templates/http/docs/service.twig

```diff
@@ -0,0 +1,41 @@
+{{ method.method | caseUpper }} {{ method.path }} HTTP/1.1
+Host: {{spec.endpoint}}
+{% for key, header in method.headers %}
+{{ key }}: {{ header }}
+{% endfor %}
+{% for key,header in spec.global.defaultHeaders %}
+{{key}}: {{header}}
+{% endfor %}
+{% for header in spec.global.headers %}
+{% if header.key != 'Mode' and header.key != 'Locale' %}
+{{header.name}}: {% for node in method.security %}{% for key,x in node|keys %}{% if x == header.key %}{{node[x]['x-appwrite']['demo']}}{% endif %}{% endfor %}{% endfor %}
+
+{% endif %}
+{% endfor %}
+
+{% for key, header in method.headers %}
+{% if header == 'application/json' %}
+{% if method.parameters.body %}
+json: {
+{% for parameter in method.parameters.body %}
+  "{{ parameter.name }}": {{ parameter | paramExample }}{% if not loop.last %},{% endif %}
+
+{% endfor %}
+}
+{% endif %}
+{% endif %}
+{% if header == 'multipart/form-data' %}
+{% set boundary = random() %}
+boundary={{boundary}}
+Content-Length: *Length of your entity body in bytes*
+{% for parameter in method.parameters.body %}
+--{{boundary}}
+Content-Disposition: form-data; name="{{parameter.name}}{% if parameter.type == "array" %}[]{% endif %}"
+{% if parameter.type != 'file' %}{% endif %}
```

Comment:
`if` and `endif` together without body? What's this doing?

Reply:
Whoops, that was left there from when I was working on the file metadata but it wasn't needed for formdata

Reply:
I'll remove it now